p2pd/main.go

```diff
@@ -37,6 +38,10 @@ func main() {
 	pubsubSignStrict := flag.Bool("pubsubSignStrict", false, "Enables pubsub strict signature verification")
 	gossipsubHeartbeatInterval := flag.Duration("gossipsubHeartbeatInterval", 0, "Specifies the gossipsub heartbeat interval")
 	gossipsubHeartbeatInitialDelay := flag.Duration("gossipsubHeartbeatInitialDelay", 0, "Specifies the gossipsub initial heartbeat delay")
+	relayEnabled := flag.Bool("relay", false, "Enables a relay")
```

Comment:
it's already enabled by default in libp2p, so this option is wrong.

Reply:
Meaning that relay will be enabled regardless of whether this option is passed.

Reply:
Switched to default to `true`